src/inc/Exceptions.hpp

```diff
@@ -47,9 +53,16 @@ namespace MSIX {
     class Win32Exception final : public Exception
```

Comment:
>class Win32Exception final : public Exception [](start = 4, length = 45)

Maybe we could guard Win32Exception and POSIXException like ThrowHrIfFalse and ThrowHrIfPOSIXFailed?

Reply:
We could but they aren't adding any real overhead except symbols.

---
In reply to: [311784039](https://github.com/Microsoft/msix-packaging/pull/195#discussion_r311784039) [](ancestors = 311784039)